dotCMS/src/main/java/com/dotcms/rendering/velocity/servlet/VelocityAdminMode.java

```diff
@@ -0,0 +1,23 @@
+package com.dotcms.rendering.velocity.servlet;
+
+import com.dotmarketing.beans.Host;
+import com.dotmarketing.business.APILocator;
+import com.liferay.portal.model.User;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+
+/**
+ * Velocity renderer for PageMode#ADMIN_MODE
```

Comment:
it seems you are introducing a new concept, explain here a bit more about the intention